finetuner/data.py

```diff
@@ -58,6 +73,34 @@ def build_dataset(
     return data
 
 
+def build_encoding_dataset(
+    model: 'InferenceEngine', data: Union[List[str], DocumentArray]
+) -> DocumentArray:
+    """If data has been provided as a list, a :class:`DocumentArray` is created
+        from the elements of the list
+
+    :param model: The InferenceEngine for the model being used for encoding
```

Comment:
dont document these functions with arguments. They are internal. Just a docstring is fine